Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -106,7 +106,8 @@ extension NavigationMapView {
         }
         
         private func updateMapOverlays(for routeProgress: RouteProgress) {
-            if routeProgress.currentLegProgress.followOnStep != nil {
+            if routeProgress.currentLegProgress.followOnStep != nil,
+               navigationMapView.routes != nil {
```

Comment:
Why do we need this additional condition? We have similar code in `CarPlayNavigationViewController`, so if it's really needed we have to update CarPlay part as well.

Reply:
Thanks for the suggestion, I'll update the CarPlay part as well.